### ISSUE TITLE
[Bug]: lots of "context deadline exceeded" error during daily regression on standlone mode 

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#20317]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Branch Name

2.0-dev

### Commit ID

c8b17ea96e2200e5ae86bde6946105171f443135

### Other Environment Information

```Markdown
- Hardware parameters: 64C 256G
- OS type:
- Others:
```

### Actual Behavior

job link:
https://github.com/matrixorigin/mo-nightly-regression/actions/runs/11974445766/job/33385610745

<img width="1415" alt="image" src="https://github.com/user-attachments/assets/826ccda3-0543-46f5-937c-6f2ce9e6b05f">

<img width="1407" alt="image" src="https://github.com/user-attachments/assets/1630dc7b-b124-4a39-a55b-f8cb76531f06">

log link:
https://shanghai.idc.matrixorigin.cn:30001/explore?panes=%7B%22GYP%22:%7B%22datasource%22:%22loki%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7Bhost%3D%5C%2210-222-1-128%5C%22,%20filename%3D%5C%22%2Fdata1%2Frunners%2Faction-runner%2F_work%2Fmo-nightly-regression%2Fmo-nightly-regression%2Fhead%2Fmo-service-c8b17ea96-20241122-222838.log%5C%22%7D%20%7C%3D%20%60context%20deadline%20exceeded%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22loki%22%7D,%22editorMode%22:%22builder%22%7D%5D,%22range%22:%7B%22from%22:%221732305822285%22,%22to%22:%221732311122629%22%7D%7D%7D&schemaVersion=1&orgId=1

### Steps to Reproduce

```Markdown
run daily regression
```

**Comment by ouyuanning** [comment]
有事务泄露。看能否拿一下这个go routine文件：ETL:/profile/CN_dd1dccb4-4d3c-41f8-b482-5251dc7a41bf_leakcheck_routine_019355c7-11fe-7468-8a0c-8958655e703b.gz

**Comment by zhangxu19830126** [comment]
fixed by #20327

**Comment by aressu1985** [comment]
fixed
